server/src/modules/repo/maps-repo.service.ts

```diff
@@ -155,6 +155,22 @@ export class MapsRepoService {
     }
     //#endregion
 
+    //#region Map Submissions
+
+    public async getSubmittedMapsSummary(submitterID: number) {
+        const submittedStatusFlags = await this.prisma.map.groupBy({
+            by: ['statusFlag'],
+            where: { submitterID: submitterID },
+            _count: {
+                statusFlag: true
+            }
+        });
+
+        return submittedStatusFlags;
```

Comment:
Best to inline return here rather than assign the prisma response to a const then immediately return. So 
```
return await this.prisma.map.groupBy({
            by: ['statusFlag'],
            where: { submitterID: submitterID },
            _count: {
                statusFlag: true
            }
        });
```